catalog/db.py

```diff
@@ -96,6 +96,35 @@ def update_column(self, column: str, value: int, id: int):
         self._connect.cursor().execute(sql)
         self._connect.commit()
 
+    def insert_into_basket(self, user_id: int, good_id: int, count: int):
+        sql1 = (f"SELECT EXISTS(SELECT 1 FROM basket "
```

Comment:
If you have more than one sql request - give them normal names

Reply:
done